fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1616,6 +1616,13 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 		                            self->toCommit.writeTypedMessage(idempotencyIdSet);
 	                            });
 
+	for (const auto& m : pProxyCommitData->idempotencyClears) {
+		auto& tags = pProxyCommitData->tagsForKey(m.param1);
+		self->toCommit.addTags(tags);
+		self->toCommit.writeTypedMessage(m);
```

Comment:
Can you add a TODO here that we need to encrypt it. I can do it after this gets merged

Reply:
Done